WordPress/gutenberg
# Site Editor: remove the "All" panel from the Template navigation sidebar.

## Description
Remove the "All" option from the Template panel of the Navigation Sidebar

Fixes #26925

## How has this been tested?
* open the Site Editor
* open the Navigation Sidebar (click on the (W) in the upper left)
* Click on "Templates"
* verify that there is no "All" option

## Screenshots 

Before:
<img width="297" alt="image" src="https://user-images.githubusercontent.com/195089/107093685-4777fd00-67cb-11eb-8947-e80c47c0b727.png">


After:
<img width="298" alt="image" src="https://user-images.githubusercontent.com/195089/107093525-ff58da80-67ca-11eb-9ef7-0ad16a64da0c.png">


## Types of changes
Breaking change (?) - all the other templates are still there.

## Checklist:
